packages/kas/src/shared-utils

```diff
@@ -0,0 +1 @@
+../../../utils/shared
```

Comment:
What does this file and line do?

Reply:
Exactly that. That's how git represents symlinks.  By using a directory that's symlinked in... we can share the code, but it's just copied into each bundle instead of being referenced as a package dependency.